package.json

```diff
@@ -4,6 +4,8 @@
   "private": true,
   "homepage": ".",
   "devDependencies": {
+    "@codemirror/language": "^0.19.5",
```

Comment:
What does this package do? Is there a reason it's in the dev dependencies?

Reply:
This package contains the class LanguageSupport to provide typing on getLanguageSupport. There is a separe types folder in https://github.com/codemirror/language/blob/main/package.json but I am unable to just install those types using e.g. yarn add -D @types/codemirror-language or yarn add -D @types/codemirror/language.